docs/Test.md

```diff
@@ -45,17 +45,41 @@ test assets, such as `describe`, `it` and `expect`:
 - `client/Hooks`
 - `client/Contexts`
 
+#### File structure
+
+TDD tests should live in a '\_\_tests\_\_' folder next to the component they are testing.
+
+- MyComponent
+  - MyComponent.ts
+  - \_\_tests\_\_
```

Comment:
out of interest, why add another directory?

Reply:
I thought it would be nice to have a clear separation of source files and test files

Reply:
IMO the .spec/.e2e serves that purpose/saves another level of directories to dive through. Thoughts @dlabaj / @nictownsend / @ajborley  ? Happy to go with the consensus (but a tweak to other docs may be needed with a new directory being added)

Reply:
I don't have strong feelings here - I typically prefer a flatter structure, but can see the merits of separate directories.